nixos/tests/systemd-cryptenroll.nix

```
@@ -2,6 +2,7 @@ import ./make-test-python.nix ({ pkgs, ... }: {
   name = "systemd-cryptenroll";
   meta = with pkgs.lib.maintainers; {
     maintainers = [ ymatsiuk ];
+    broken = true; # times out after two hours
```

```suggestion
    broken = true; # times out after two hours, details -> https://github.com/NixOS/nixpkgs/issues/167994
```

Sorry about that, somewhat I thought it would be easier to fix and never considered the option of marking the test as broken.